ts/features/bonus/cgn/store/actions/details.ts

```diff
@@ -0,0 +1,13 @@
+import { ActionType, createAsyncAction } from "typesafe-actions";
+import { CgnStatus } from "../../../../../../definitions/cgn/CgnStatus";
+
+/**
+ * get and handle activation state of a CGN
+ */
+export const cgnDetails = createAsyncAction(
+  "CGN_DETAILS_REQUEST",
+  "CGN_DETAILS_SUCCESS",
+  "CGN_DETAILS_FAILURE"
+)<void, CgnStatus, Error>();
```

Comment:
```suggestion
)<void, CgnStatus, NetworkError>();
```
it could be useful to catch also timeout errors
If you think it could be good, you should use `getNetworkError` util function

Reply:
Nice suggestion, just didn't know but can be useful 

added in a9ed676